apmpackage/apm/changelog.yml

```diff
@@ -3,6 +3,11 @@
     - description: Placeholder
       type: enhancement
       link: https://github.com/elastic/apm-server/pull/123
+- version: "8.10.1"
```

Comment:
I think we can just skip the entry if there's nothing interesting.

Reply:
Fixed